scimma_admin/scimma_admin/settings.py

```diff
@@ -118,24 +119,22 @@ def get_rds_db(db_instance_id):
     }
 
 
-# Password validation
-# https://docs.djangoproject.com/en/3.0/ref/settings/#auth-password-validators
+# Authentication
+# https://mozilla-django-oidc.readthedocs.io/en/stable/settings.html
 
-AUTH_PASSWORD_VALIDATORS = [
-    {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
-    },
-]
+AUTHENTICATION_BACKENDS = (
+    'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
+)
+OIDC_RP_CLIENT_ID = "cilogon:/client_id/29ad6d4e4198bd207e916128fec3c656"
+OIDC_RP_CLIENT_SECRET = get_secret("scimma-admin-cilogon-client-secret")
```

Comment:
This bit is application-specific, and you get the values from Warren. Maybe I should add a comment to that effect in settings.py, but I'm at least noting it in the PR for future readers.